source/kraken/make_disruption_from_chaos.cpp

```diff
@@ -321,6 +321,30 @@ make_impact(const chaos::Impact& chaos_impact, nt::PT_Data& pt_data,
     return impact;
 }
 
+bool is_publishable(transit_realtime::TimeRange publication_period,
```

Comment:
It will be very easy to add some unittest on this function additionally to the integration tests

Reply:
done